src/main/java/com/cinema/dao/impl/CinemaHallDaoImpl.java

```diff
@@ -21,6 +22,7 @@ public CinemaHall add(CinemaHall cinemaHall) {
             transaction = session.beginTransaction();
             session.save(cinemaHall);
             transaction.commit();
+            Application.logger.info(cinemaHall);
```

Comment:
```suggestion
            Application.logger.info("Successfully added " + cinemaHall + " to DB");
```
You can also create a `Logger` for each Dao without importing `Application`. If you have a problem running the code then, you can replace the old injector with the new one (see the comments for today's Q&A with Ksenia)

Reply:
I totally agree
please do not import Logger - create it in each DaoImpl class
use the updated Injector if needed